libraries/AP_Scripting/examples/CAN_read.lua

```diff
@@ -10,6 +10,11 @@ if not driver1 and not driver2 then
    return
 end
 
+-- Only accept DroneCAN node status msg on second driver
+-- node status is message ID 341
+-- Message ID is 16 bits left shifted by 8 in the CAN frame ID.
+driver2:add_filter(0x00FFFF00, uint32_t(341) << 8)
```

Comment:
This might be slightly easier to understand. Or at least it matches the comment better.....
```suggestion
driver2:add_filter(uint32_t(0xFFFF) << 8, uint32_t(341) << 8)
```